deepreg/model/layer_util.py

```diff
@@ -369,63 +368,6 @@ def warp_grid(grid: tf.Tensor, theta: tf.Tensor) -> tf.Tensor:
     return grid_warped
 
 
-def gaussian_filter_3d(kernel_sigma: Union[Tuple, List]) -> tf.Tensor:
```

Comment:
I would keep this func and use it for unit-testing, as the separable conv should be identical to the conv3d option? can we test like this?